pkg/detector/detector.go

```diff
@@ -274,18 +273,7 @@ func (d *ResourceDetector) Reconcile(key util.QueueKey) error {
 // the specified apis will be ignored as well.
 //
 // If '--skipped-propagating-namespaces' is specified, all APIs in the skipped-propagating-namespaces will be ignored.
-func (d *ResourceDetector) EventFilter(obj interface{}) bool {
-	key, err := ClusterWideKeyFunc(obj)
-	if err != nil {
-		return false
-	}
-
-	clusterWideKey, ok := key.(keys.ClusterWideKey)
-	if !ok {
-		klog.Errorf("Invalid key")
-		return false
-	}
-
+func (d *ResourceDetector) SkippedFromPropagating(clusterWideKey keys.ClusterWideKey) bool {
```

Comment:
After renaming the function, the implementation should be updated as well, when a resource is skipped, it should return `true` now.

Reply:
Fixed, thanks for elaborating!